newsplease/pipeline/pipelines.py

```diff
@@ -570,6 +571,147 @@ def process_item(self, item, spider):
                 return item
 
 
+class ContentFilter(object):
+
+    class BiOp(object):
```

Comment:
Could you please use a better understandable class name?

Reply:
and briefly state what this class is doing

Reply:
have added the comments